lib/getPostcssResult.js

```diff
@@ -1,5 +1,11 @@
 /* @flow */
 "use strict"
+
+/*:: type PostcssOptions = {
```

Comment:
This is the right idea, I think. Did you see that we're defining these `type`s in declaration files? The PostCSS one is https://github.com/stylelint/stylelint/blob/master/decls/postcss.js.